code/modules/antagonists/revenant/revenant_abilities.dm

```diff
@@ -35,9 +35,12 @@
 		if(target.ckey)
 			to_chat(src, "<span class='revennotice'>[target.p_their(TRUE)] soul burns with intelligence.</span>")
 			essence_drained += rand(20, 30)
-		if(target.stat != DEAD)
+		if(target.stat != DEAD && !HAS_TRAIT(target, TRAIT_WEAK_SOUL))
 			to_chat(src, "<span class='revennotice'>[target.p_their(TRUE)] soul blazes with life!</span>")
 			essence_drained += rand(40, 50)
+		if(HAS_TRAIT(target, TRAIT_WEAK_SOUL) && !target.ckey)
+			to_chat(src, "<span class='revennotice'>[target.p_their(TRUE)] soul is weak and underdeveloped. They won't be worth very much.</span>")
+			essence_drained = 5
 		else
 			to_chat(src, "<span class='revennotice'>[target.p_their(TRUE)] soul is weak and faltering.</span>")
```

Comment:
okay, hear me out here:
```suggestion
			if(target.stat != DEAD)
				to_chat(src, "<span class='revennotice'>[target.p_their(TRUE)] soul blazes with life!</span>")
				essence_drained += rand(40, 50)
		else
			to_chat(src, "<span class='revennotice'>[target.p_their(TRUE)] soul is weak and faltering.</span>")
```
yes, the added indentation there is intentional

Reply:
otherwise, you can just have a mortal minion mutadone smoke a shitton of monkeys for you and basically still pull off this trick

Reply:
I sorta did want to leave that as a thing since lol who would do that?

But then again maybe it's best to reward consuming the souls of actual players.